tests/_test_mscolab/test_file_manager.py

```diff
@@ -79,7 +79,7 @@ def test_modify_user(self):
             self.fm.modify_user(user_query, attribute="confirmed", value=True)
             user_query = User.query.filter_by(id=user.id).first()
             assert user_query.confirmed is True
-            assert user_query.confirmed_on.replace(tzinfo=None) == confirm_time.replace(tzinfo=None)
+            assert user_query.confirmed_on.replace(tzinfo=datetime.timezone.utc) == confirm_time
```

Comment:
```suggestion
            assert user_query.confirmed_on == confirm_time
```

I think the query itself should already return an aware datetime object. From what I've read this might require a custom column type in the database model.

Reply:
@matrss Can you please elaborate on this

Reply:
What I meant is exactly what you found yourself already: defining a custom type by augmenting the standard DateTime type with a conversion to UTC and interpreting the value in that database column as UTC. It is documented more in-depth here: https://docs.sqlalchemy.org/en/20/core/custom_types.html#augmenting-existing-types